sqlglot/expressions.py

```diff
@@ -5745,6 +5745,10 @@ class Upper(Func):
     _sql_names = ["UPPER", "UCASE"]
 
 
+class PearsonCorrelation(Binary, AggFunc):
```

Comment:
can you just call this Corr?

Reply:
the can remove _sql_names

Reply:
There are [other forms](https://docs.oracle.com/en/database/oracle/oracle-database/19/sqlrf/CORR_A.html#GUID-B2DED35A-2ECE-4DF0-BDA4-28F28B7BCA23) of correlations though, figured we'd be more explicit just in case.